packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -105,7 +121,8 @@ contract L1Block is ISemver, IGasToken {
         uint64 _sequenceNumber,
         bytes32 _batcherHash,
         uint256 _l1FeeOverhead,
-        uint256 _l1FeeScalar
+        uint256 _l1FeeScalar,
+        bool _isDeposit
```

Comment:
We should remove the `_isDeposit` argument here, there should be 0 diff to `setL1BlockValues`